addons/l10n_it_edi/tests/test_edi_export.py

```diff
@@ -304,6 +304,29 @@ def setUpClass(cls):
         cls.zero_tax_invoice._post()
         cls.negative_price_invoice._post()
         cls.negative_price_credit_note._post()
+    
+    
+    def test_export_zero_amount_move(self):
+        """ When a move has an amount of 0, a float division by zero error is triggered.
+        """
+        usd_currency = self.env.ref('base.USD')
+        usd_currency.active = True
+        zero_amount_move = self.env['account.move'].with_company(self.company).create({
+            'move_type': 'out_invoice',
+            'invoice_date': datetime.date(2022, 3, 24),
+            'invoice_date_due': datetime.date(2022, 3, 24),
+            'partner_id': self.italian_partner_no_address_codice.id,
+            "currency_id": usd_currency.id,
+            'invoice_line_ids': [
+                (0, 0, {
+                    "amount_currency": 0,
+                    "debit": 0,
+                    "credit": 0
+                    }),
+                ],
+            })
+        zero_amount_move._prepare_fatturapa_export_values()
+        self.assertTrue(True)
```

Comment:
```suggestion
        self.assertEqual(len(zero_amount_move.invoice_line_ids), 1)
        self.assertEqual(zero_amount_move.invoice_line_ids[0].amount_currency, 0)
```
Maybe we could at least check a few values?